network/vpc/how-to/create-private-network.mdx

```diff
@@ -1,57 +1,108 @@
+Private Networks also have built-in **D**ynamic **H**ost **C**onfiguration **P**rotocol (DHCP), making it easy to manage the private IP addresses of your resources on the network. IPs are staticallly allocated when resources are attached to a Private Network, and are managed by our internal [IPAM](/newtork/vpc/concepts/#ipam)
+
+<Message type="note">
+Previously, Private Networks at Scaleway were zoned. Only resources from within one defined AZ could be attached to each network. Now, all Private Networks are regional, and resources from any AZ within that network's region can be attached. "Old" zoned Private Networks have all been automatically migrated to become regional.
+
+While DHCP is built into all new Private Networks, it may not be automatically activated for older Private Networks. Check our [migration](/network/vpc/reference-content/vpc-migration/) documentation for more information.
+</Message>
 
 <Macro id="iam-requirements" />
  
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
 </Message>
 
-## How to create a regional Private Network
+## How to create a Private Network
 
 1. Click **VPC** in the **Network** section of the side menu.
 
-    A default VPC for each available region has been automatically created for you.
-
     <Lightbox src="scaleway-vpc.webp" alt="" />
+    TODO: check screenshot
+
+    Each Private Network must be created within a VPC. 
+
+2. Click the VPC you wish to create your Private Network in.
 
-2. Click the **Default VPC** for your chosen region.
+3. Click the **Private Networks** tab.
 
-    If you already have existing regional Private Networks in this VPC, a list of these displays. Otherwise, you see a welcome screen.
+    If you already have existing Private Networks in this VPC, a list of these displays. Otherwise, you see a welcome screen.
 
     <Lightbox src="scaleway-reg-pn.webp" alt="" />
+    TODO: check screenshot
 
-3. Click **+ Create regional Private Network**. The creation screen displays.
+4. Click **+ Create Private Network**. The creation screen displays.
 
     <Lightbox src="scaleway-create-rpn.webp" alt=""/>
+    TODO: check screenshot.
+
+5. Enter a **name** for the network, or leave the randomly-generated name in place. Optionally, you can also add **tags** to help organize your Private Networks. Each tag should be separated by a space.
+
+6. **Optional**: Click **Advanced Settings** to edit these settings, otherwise they will be left at default recommended values. The only advanced setting available concerns the creation of an IPv4 CIDR block for the Pirvate Network, from which private IP addresses will be allocated. You cannot change the CIDR block after the Private Network has been created. See the section below on [configuring CIDR](#how-to-configure-cidr) for more help with choosing these settings.
```

Comment:
```suggestion
6. **Optional**: Click **Advanced Settings** to edit these settings, otherwise they will be left at default recommended values. The only advanced setting available concerns the creation of an IPv4 CIDR block for the Private Network, from which private IP addresses will be allocated. You cannot change the CIDR block after the Private Network has been created. See the section below on [configuring CIDR](#how-to-configure-cidr) for more help with choosing these settings.
```